requirements.txt

```diff
@@ -2,9 +2,9 @@ numpy>=1.11
 sympy==1.1
 scipy
 matplotlib
-pytest
+pytest-runner
 flake8>=2.1.0
-parso==0.1.0
+jedi
```

Comment:
???

Reply:
`jedi` gets install by something else and creates conflicts with `parso==0.1.0` as it already installs it with a different version

Reply:
I didn't get it sorry. We're dropping ``parso`` here correct? who's using ``jedi``? if you don't install it, what does it fail?

Reply:
>who's using jedi?
It's a requirement for one of the other requirements, not sure which

Reply:
`error: parso 0.1.0 is installed but parso>=0.2.0 is required by {'jedi'}`

Still no idea where this `jedi` is coming from but crashing the install.

Reply:
ahhhhhh! Then what I don't understand is why you dropped ``parso==0.1.0``? I'm concerned we are installing ``jedi`` only to indirectly get ``parso``. Ie. we're not using jedi at all, but only one of its dependencies

Reply:
I'm still trying to figure out where this `jedi` is coming from. These kind of dependencies never show up with `requirement.txt` as it installs all requires separately while `pip` creates an `easy_install` that checks all cross-dependnecies

Reply:
if you put back ``parso==0.1.0`` and remove ``jedi``, does it work?